### DISCUSSION TITLE
Unescaping fields in json log lines

### DISCUSSION THREAD

**Discussion by igorcoding** [grafana/grafana#42050] [Observability]
Hello!

We are trying to view logs from packetbeat in Grafana (using Loki) and facing with a display issue of some fields.

The log line is a raw json string which looks like the following:
![image](https://user-images.githubusercontent.com/5521116/142847220-f1e6885d-ec86-4abf-8705-c42a97073433.png)

And the fields we are interested in:
![image](https://user-images.githubusercontent.com/5521116/142847298-fb8c0c25-0c2b-4422-8931-fd87d3f8a66c.png)

We are intercepting http traffic and trying to see all http requests/responses within grafana. The problem is that these fields are not usable because of the new line escaping. We are trying to migrate from ELK to Loki and it will be great if Grafana could display these fields properly without escaping. Actually this is also valid for fields like stacktraces which are sent currently also as separate fields in json logs. 
A possible solution could be to have a button for each field that would turn off/on escaping.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
